app/src/main/java/com/github/se/eduverse/ui/timetable/DetailsEventScreen.kt

```diff
@@ -63,18 +59,7 @@ fun DetailsEventScreen(
   }
 
   Scaffold(
-      topBar = {
-        TopNavigationBar(event.name, navigationActions) {
-          IconButton(
-              onClick = {
-                timeTableViewModel.deleteScheduled(event)
-                navigationActions.goBack()
-              },
-              modifier = Modifier.testTag("deleteButton")) {
-                Icon(Icons.Default.Delete, contentDescription = "Delete")
-              }
-        }
-      },
+      topBar = { TopNavigationBar(navigationActions, screenTitle = null) },
```

Comment:
Here you shouldn't have remove the delete button. You can copy-paste this after TopNavigationBar(...) :
```
{
          IconButton(
              onClick = {
                timeTableViewModel.deleteScheduled(event)
                navigationActions.goBack()
              },
              modifier = Modifier.testTag("deleteButton")) {
                Icon(Icons.Default.Delete, contentDescription = "Delete")
              }
        }
```

Reply:
Ok, good remark, I will do that